source/docs/articles/sites/code/pantheon-yml.md

```diff
@@ -0,0 +1,48 @@
+---
+title: The pantheon.yml Configuration File
+description: Developers can set project configuration in a pantheon.yml file stored in the root of their code repository.  
+---
+
+Configuration stored in [YAML](https://en.wikipedia.org/wiki/YAML) files has become a standard across PAAS providers. The `pantheon.yml` file allows developers to hook into platform workflows. It will soon give developers the ability to configure other settings for their sites.
+
+## About `pantheon.yml`
+The file can contain two collections, `api version:` and `workflows:`.  
+
+### API Version
+The current API version for pantheon.yml is `1`. This is the only valid value to place in this field. Additional valid values may be introduced in the future.
+```yaml
+api_version: 1
+```
+### Workflows 
+The workflows collection allows developers to hook into platform operations. Developers Identify the workflows they want to hook into, the workflow “stage” (before or after) to hook into, and then define a sequence of mapping blocks that define the operations the platform will run during the workflow stage. For more information, see [Quicksilver Platform Hooks](/docs/articles/sites/quicksilver/). 
```

Comment:
yeah.. our in-site docs links are relative to the production output state, not the source repository state. We have a symlink in production, for images, that would make these links work in both places, if we were to change all docslinks from `docs/articles...` to `source/docs/articles`. I'm glad you found it.